modules/docs/mdx/versionsTable.tsx

```diff
@@ -4,40 +4,6 @@ import {StatusIndicator, StatusIndicatorVariant} from '@workday/canvas-kit-previ
 // @ts-ignore: Cannot find module error
 import {version} from '../../../lerna.json';
 
-// When we release a new version, add the support version before the first item.
-const allVersions = [
-  {
-    versionNumber: version, // This will always be the current major version
-    documentation: 'https://github.com/Workday/canvas-kit',
-  },
-  {
-    versionNumber: 11, // This is support, update this when we release v13
-    documentation: 'https://d2krrudi3mmzzw.cloudfront.net/v11/?path=/docs/welcome--page',
-  },
-  {
-    versionNumber: 10,
-    documentation: 'https://d2krrudi3mmzzw.cloudfront.net/v10/?path=/docs/welcome--page',
-  },
-  {
-    versionNumber: 9,
-    documentation: 'https://d2krrudi3mmzzw.cloudfront.net/v9/?path=/docs/welcome--page',
-  },
-  {
-    versionNumber: 8,
-    documentation: 'https://d2krrudi3mmzzw.cloudfront.net/v8/?path=/docs/welcome--page',
-  },
-  {
-    versionNumber: 7,
-    documentation:
-      'https://d2krrudi3mmzzw.cloudfront.net/v7/?path=/story/welcome-getting-started--page',
-  },
-  {
-    versionNumber: 6,
-    documentation:
-      'https://d2krrudi3mmzzw.cloudfront.net/v7/?path=/story/welcome-getting-started--page',
-  },
-];
-
```

Comment:
this was annoying so i rewrote it to just use the version site of the site. Hopefully we don't need to update this anymore.

It would be nice to have a function that get the current major version of CK...